src/nreplClient.ts

```diff
@@ -42,31 +42,39 @@ interface nREPLCloseMessage {
     op: string;
 }
 
-const complete = (symbol: string, ns: string, callback): void => {
+const complete = (symbol: string, ns: string): Promise<any> => {
     const msg: nREPLCompleteMessage = { op: 'complete', symbol, ns };
-    send(msg).then(respObjs => callback(respObjs[0]));
+    return send(msg).then(respObjs => respObjs[0]);
 };
 
-const info = (symbol: string, ns: string, callback) => {
+const info = (symbol: string, ns: string): Promise<any> => {
     const msg: nREPLInfoMessage = { op: 'info', symbol, ns };
-    send(msg).then(respObjs => callback(respObjs[0]));
+    return send(msg).then(respObjs => respObjs[0]);
 };
 
-const evaluate = (code: string): Promise<any[]> => clone().then((new_session) => {
-    const session_id = new_session['new-session'];
-    const msg: nREPLSingleEvalMessage = { op: 'eval', code: code, session: session_id };
-    return send(msg);
-});
+const evaluate = (code: string): Promise<any[]> => {
+    return clone().then((new_session) => {
+        const session_id = new_session['new-session'];
+        const msg: nREPLSingleEvalMessage = { op: 'eval', code: code, session: session_id };
+        return send(msg);
+    });
+};
 
-const evaluateFile = (code: string, filepath: string): Promise<any[]> => clone().then((new_session) => {
-    const session_id = new_session['new-session'];
-    const msg: nREPLEvalMessage = { op: 'load-file', file: code, 'file-path': filepath, session: session_id };
-    return send(msg);
-});
+const evaluateFile = (code: string, filepath: string): Promise<any[]> => {
+    return clone().then((new_session) => {
+        const session_id = new_session['new-session'];
+        const msg: nREPLEvalMessage = { op: 'load-file', file: code, 'file-path': filepath, session: session_id };
+        return send(msg);
+    });
+};
 
-const stacktrace = (session: string): Promise<any> => send({ op: 'stacktrace', session: session });
+const stacktrace = (session: string): Promise<any> => {
+    return send({ op: 'stacktrace', session: session });
+}
```

Comment:
I just changed it to be consistent with the other lines in this file, where a bunch of the other send functions have the body of the function in a block on a new line.

But I don't feel strongly about this change at all. Actually, looking back on the original state, it seems like there is some consistency that I missed in the first place - that if it's an easy one-liner, it's done as a one-liner style, but if it wouldn't fit on one line, it's done on two lines. So maybe this was unnecessary :-)